src/components/API-calls.js

```diff
@@ -4,8 +4,26 @@ const api = axios.create({
   baseURL: "https://my-backend-server-josh.herokuapp.com",
 });
 
-export const getReviewList = () => {
-  return api.get(`/api/reviews`).then(({ data }) => {
+export const getReviewList = (AppendUrl) => {
+  return api.get(`/api/reviews${AppendUrl}`).then(({ data }) => {
```

Comment:
I'd look at using the params object with axios, as it will add them ?queryHere bit for you